packages/material-ui/test/typescript/components.spec.tsx

```diff
@@ -1048,7 +1048,7 @@ const PopoverTest = () => <Popover open ModalClasses={{ root: 'foo', hidden: 'ba
 
 const InputLabelTest = () => (
   <InputLabel
-    FormLabelClasses={{
+    classes={{
       root: 'foo',
       asterisk: 'foo',
```

Comment:
@pelotom This was accepted **before** I added the `asterisk` to the `InputLabelClassKey`. Hovering over the `classes` prop did show `Partial<Record<InputLabelClassKey, string>>` which should reject that. Writing 
`const classes: Partial<Record<InputLabelClassKey, string>> = { asterisk: 'foo' }` did however throw an error.

I suspect this is a typescript bug?